assets/theme-css/figure.css

```diff
@@ -0,0 +1,43 @@
+img {
+  max-width: 100%;
+}
+
+img.align-left,
```

Comment:
Heh, on how many elements will we use this `align-left` class?!

Reply:
I just moved this out of styles.css. The only new thing in this file is
```
strong.caption-title {
  font-weight: bold;
}
```
The rest came from PST dependencies, I believe. I will move it back for now and then investigate where it came from and move it to a different location to make it clear.

Reply:
I removed the `figure.css` and added the new styling to `styles.css`.

Reply:
We can look at `align-left` in a future PR.